package.json

```diff
@@ -118,7 +118,7 @@
     "typescript": "3.5.3",
     "uglify-js": "mishoo/UglifyJS2#harmony-v2.8.22",
     "vsce": "^1.58.0",
-    "vscode": "^1.1.36",
+    "vscode": "^1.1.33",
```

Comment:
why are we doing this?

Reply:
We upgraded it earlier, just reverting it since it's not necessary. I can let it be too.